### ISSUE TITLE
Tests don't work anymore

### ISSUE THREAD

**Issue by danielpanteleit** [tomav/docker-mailserver#1147]
When running the tests with a fresh build, the tests hang in the mail sending section:

```
docker exec mail addmailuser pass@localhost.localdomain 'may be \a `p^a.*ssword'
sleep 10
# Setup sieve & create filtering folder (INBOX/spam)
docker cp "`pwd`/test/config/sieve/dovecot.sieve" mail:/var/mail/localhost.localdomain/user1/.dovecot.sieve
docker exec mail /bin/sh -c "maildirmake.dovecot /var/mail/localhost.localdomain/user1/.INBOX.spam"
docker exec mail /bin/sh -c "chown 5000:5000 -R /var/mail/localhost.localdomain/user1/.INBOX.spam"
sleep 30
# Sending test mails
docker exec mail /bin/sh -c "nc 0.0.0.0 25 < /tmp/docker-mailserver-test/email-templates/amavis-spam.txt"
220 mail.my-domain.com ESMTP Postfix (Debian)
```

One thing I found out is, that the generated accounts are now broken because of error messages that are also placed in the accounts file (file `test/config/postfix-accounts.cf`, generated in the first line of generate-accounts):

```
doveconf: Warning: NOTE: You can get a new clean config file with: doveconf -Pn > dovecot-new.conf
doveconf: Warning: Obsolete setting in /etc/dovecot/conf.d/10-ssl.conf:46: ssl_dh_parameters_length is no longer needed
doveconf: Warning: Obsolete setting in /etc/dovecot/conf.d/10-ssl.conf:49: ssl_protocols has been replaced by ssl_min_protocol
user1@localhost.localdomain|{SHA512-CRYPT}$6$blOnXix2bUnfkPA0$RAQ25hCzmgxEmdAoZ/foo35po42vu8UXmKEWdJCEQhpzMrThaVAQFCNh4Tjmlb0WAKPb9vyeU477k581If2Sm1
doveconf: Warning: NOTE: You can get a new clean config file with: doveconf -Pn > dovecot-new.conf
doveconf: Warning: Obsolete setting in /etc/dovecot/conf.d/10-ssl.conf:46: ssl_dh_parameters_length is no longer needed
doveconf: Warning: Obsolete setting in /etc/dovecot/conf.d/10-ssl.conf:49: ssl_protocols has been replaced by ssl_min_protocol
user2@otherdomain.tld|{SHA512-CRYPT}$6$KVJj2Bd2g3hiW.V3$9PcMTnig4pPeJWDIw4TjXGpDSvRqveBNXWabAL42qXnyZi.p0hcF7M6ugLeVOx2VHkWGjz7Llwmm7cwiElo4L.
doveconf: Warning: NOTE: You can get a new clean config file with: doveconf -Pn > dovecot-new.conf
doveconf: Warning: Obsolete setting in /etc/dovecot/conf.d/10-ssl.conf:46: ssl_dh_parameters_length is no longer needed
doveconf: Warning: Obsolete setting in /etc/dovecot/conf.d/10-ssl.conf:49: ssl_protocols has been replaced by ssl_min_protocol
added@localhost.localdomain|{SHA512-CRYPT}$6$0gQ/i5oH5xaKqCOc$NFvHcGkERoOkZkuE9bdJbrk9zC.R8joiJAcrAj0XAJu/aE98M6wg9RDnigaxsTy/ryu56qSlrGgmxmwlxPvED/
pass@localhost.localdomain|{SHA512-CRYPT}$6$83nfp60G1FLZelHk$2rrg9rmrxIb/qr2xRKfZlnKvaMonSWVvn5OcAw/Yeqj5LbGx1DmQNLC6/oKNeV.1z3af/rUcjjT5AOsVmUEL8.
```

There are also multiple errors/warnings in the log files:

```
Mar 23 19:50:46 mail postfix/postscreen[3261]: CONNECT from [127.0.0.1]:43152 to [127.0.0.1]:25
Mar 23 19:50:46 mail postfix/postscreen[3261]: WHITELISTED [127.0.0.1]:43152
Mar 23 19:50:46 mail postfix/smtpd[3262]: warning: /etc/postfix/vmailbox, line 2: record is in "key: value" format; is this an alias file?
Mar 23 19:50:46 mail postfix/smtpd[3262]: warning: /etc/postfix/vmailbox, line 3: record is in "key: value" format; is this an alias file?
Mar 23 19:50:46 mail postfix/smtpd[3262]: error: /etc/postfix/vmailbox, line 3: duplicate entry: "doveconf:"
Mar 23 19:50:46 mail postfix/smtpd[3262]: connect from localhost[127.0.0.1]
Mar 23 19:50:46 mail opendmarc[291]: ignoring connection from localhost
Mar 23 19:50:46 mail postfix/smtpd[3262]: improper command pipelining after HELO from localhost[127.0.0.1]: MAIL FROM: spam@external.tld\nRCPT TO: user1@localhost.localdomain\nDATA\nFrom: Docker Mail Server <doc
Mar 23 19:50:46 mail dovecot: auth: Error: passwd-file /etc/dovecot/userdb: User doveconf has invalid UID ' NOTE'
Mar 23 19:50:46 mail dovecot: auth: Error: passwd-file /etc/dovecot/userdb: User doveconf has invalid UID ' Obsolete setting in /etc/dovecot/conf.d/10-ssl.conf'
Mar 23 19:50:46 mail dovecot: auth: Error: passwd-file /etc/dovecot/userdb: User doveconf has invalid UID ' Obsolete setting in /etc/dovecot/conf.d/10-ssl.conf'
Mar 23 19:50:46 mail dovecot: auth: Error: passwd-file /etc/dovecot/userdb: User doveconf has invalid UID ' NOTE'
Mar 23 19:50:46 mail dovecot: auth: Error: passwd-file /etc/dovecot/userdb: User doveconf has invalid UID ' Obsolete setting in /etc/dovecot/conf.d/10-ssl.conf'
Mar 23 19:50:46 mail dovecot: auth: Error: passwd-file /etc/dovecot/userdb: User doveconf has invalid UID ' Obsolete setting in /etc/dovecot/conf.d/10-ssl.conf'
Mar 23 19:50:46 mail dovecot: auth: Error: passwd-file /etc/dovecot/userdb: User doveconf has invalid UID ' NOTE'
Mar 23 19:50:46 mail dovecot: auth: Error: passwd-file /etc/dovecot/userdb: User doveconf has invalid UID ' Obsolete setting in /etc/dovecot/conf.d/10-ssl.conf'
Mar 23 19:50:46 mail dovecot: auth: Error: passwd-file /etc/dovecot/userdb: User doveconf has invalid UID ' Obsolete setting in /etc/dovecot/conf.d/10-ssl.conf'
Mar 23 19:50:46 mail postfix/trivial-rewrite[3266]: fatal: open dictionary: expecting "type:name" form instead of "doveconf:"
Mar 23 19:50:47 mail postfix/master[3126]: warning: process /usr/lib/postfix/sbin/trivial-rewrite pid 3266 exit status 1
Mar 23 19:50:47 mail postfix/master[3126]: warning: /usr/lib/postfix/sbin/trivial-rewrite: bad command startup -- throttling
Mar 23 19:51:47 mail postfix/trivial-rewrite[3474]: fatal: open dictionary: expecting "type:name" form instead of "doveconf:"
Mar 23 19:51:48 mail postfix/master[3126]: warning: process /usr/lib/postfix/sbin/trivial-rewrite pid 3474 exit status 1
Mar 23 19:51:48 mail postfix/master[3126]: warning: /usr/lib/postfix/sbin/trivial-rewrite: bad command startup -- throttling
Mar 23 19:52:48 mail postfix/trivial-rewrite[3684]: fatal: open dictionary: expecting "type:name" form instead of "doveconf:"
Mar 23 19:52:49 mail postfix/master[3126]: warning: process /usr/lib/postfix/sbin/trivial-rewrite pid 3684 exit status 1
Mar 23 19:52:49 mail postfix/master[3126]: warning: /usr/lib/postfix/sbin/trivial-rewrite: bad command startup -- throttling
Mar 23 19:53:49 mail postfix/trivial-rewrite[3889]: fatal: open dictionary: expecting "type:name" form instead of "doveconf:"
Mar 23 19:53:50 mail postfix/master[3126]: warning: process /usr/lib/postfix/sbin/trivial-rewrite pid 3889 exit status 1
Mar 23 19:53:50 mail postfix/master[3126]: warning: /usr/lib/postfix/sbin/trivial-rewrite: bad command startup -- throttling
Mar 23 19:54:50 mail postfix/trivial-rewrite[4097]: fatal: open dictionary: expecting "type:name" form instead of "doveconf:"
Mar 23 19:54:51 mail postfix/master[3126]: warning: process /usr/lib/postfix/sbin/trivial-rewrite pid 4097 exit status 1
Mar 23 19:54:51 mail postfix/master[3126]: warning: /usr/lib/postfix/sbin/trivial-rewrite: bad command startup -- throttling
```

The problem should also occur on a new travis build.

**Comment by johansmitsnl** [comment]
Could you fix it and provide a pr?

**Comment by erik-wramner** [comment]
I fixed the broken unit tests in pull request #1177.

**Comment by tomav** [comment]
Fixed.
